src/main/scala/cromwell/engine/workflow/WorkflowActor.scala

```diff
@@ -50,9 +50,14 @@ object WorkflowActor {
   final case class CachesCreated(startMode: StartMode) extends WorkflowActorMessage
   final case class AsyncFailure(t: Throwable) extends WorkflowActorMessage
   final case class PerformTransition(toState: WorkflowState) extends WorkflowActorMessage
+  sealed trait PersistenceMessage extends WorkflowActorMessage {
+    def callKey: ExecutionStoreKey
+    def executionStatus: ExecutionStatus
+  }
   final case class PersistenceCompleted(callKey: ExecutionStoreKey,
```

Comment:
Yay ADT!  This makes me think what is currently the `PersistenceCompleted` message should probably be renamed to `PersistenceSucceeded` to be more clear.  There is also a multi-persistence `PersistencesCompleted` (note the plural) that might be renamed and brought under the base type as well.